### PR TITLE
release fmlib 0.5.2

### PR DESCRIPTION
**Pull request by hbr**
new file:   packages/fmlib/fmlib.0.5.2/opam
	new file:   packages/fmlib_js/fmlib_js.0.5.2/opam
	new file:   packages/fmlib_parse/fmlib_parse.0.5.2/opam
	new file:   packages/fmlib_pretty/fmlib_pretty.0.5.2/opam
	new file:   packages/fmlib_std/fmlib_std.0.5.2/opam

### CODE REVIEW

File: packages/fmlib_parse/fmlib_parse.0.5.2/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+]
+
+
+depends: [
+    "ocaml" {>= "4.08.0"}
+    "dune" {>= "1.10"}
+    "odoc" {with-doc}
+    "ppx_inline_test" {>= "v0.13.0"}
+    "fmlib_std"    {>= "0.5.0"}
+    "fmlib_pretty" {>= "0.5.0"}
```
Comment (raphael-proust) [inline packages/fmlib_parse/fmlib_parse.0.5.2/opam:39]: Any reason to not use `=version` here?

Comment (hbr) [reply packages/fmlib_parse/fmlib_parse.0.5.2/opam:39]: What does `=version` mean exactly?

Comment (kit-ty-kate) [reply packages/fmlib_parse/fmlib_parse.0.5.2/opam:39]: it means those packages should stay in sync with one another (fmlib_parse.0.5.2 requires fmlib_std.0.5.2). You've used it on all your other packages

Comment (hbr) [reply packages/fmlib_parse/fmlib_parse.0.5.2/opam:39]: Thanks for the explanation. Then I think `=version` is better. Is it necessary to change it for this release or is it sufficient if I change it for the next release?

**Comment by raphael-proust** [comment]
LGTM now. The remaining failures seem to be internal to the runners.

**Comment by dinosaure** [comment]
Errors are about the experimental support on MacOS. Let's merge. Thanks!

**Pull request merged**